spec/podfile/dsl_spec.rb

```diff
@@ -679,6 +679,102 @@ module Pod
 
     #-------------------------------------------------------------------------#
 
+    describe 'ensure_bundler!' do
+      def stash_bundler_environment
+        @stored_bundle_gemfile = ENV['BUNDLE_GEMFILE']
+        @stored_bundle_bin_path = ENV['BUNDLE_BIN_PATH']
+        @stored_bundle_version = ENV['BUNDLER_VERSION']
+      end
+
+      def restore_bundler_environment
+        ENV['BUNDLE_BIN_PATH'] = @stored_bundle_bin_path
+        ENV['BUNDLE_GEMFILE'] = @stored_bundle_gemfile
+        ENV['BUNDLER_VERSION'] = @stored_bundle_version
+      end
+
+      it 'Prints warning when Bundler environment not found' do
```

Comment:
one more nit: For all those `it` here can you do lower case?

Reply:
✔️